File: tests/fixtures/csharp/UsersController.cs
using Microsoft.AspNetCore.Mvc;

namespace Demo.Controllers
{
    [ApiController]
    public class UsersController : ControllerBase
    {
        [HttpGet("/users/{id}")]
        public User GetUser(int id)
        {
            return _repository.FindById(id);
        }

        [HttpPost("/users")]
        public User CreateUser(User user)
        {
            var sql = "INSERT INTO users (name) VALUES (@name)";
            _db.Execute(sql, user);
            return user;
        }
    }
}

File: tests/fixtures/ruby/users_controller.rb
class UsersController < ApplicationController
  def show
    @user = User.find(params[:id])
    render json: @user
  end

  def create
    ActiveRecord::Base.connection.execute("INSERT INTO users (name) VALUES ('x')")
  end
end

get '/users/:id', to: 'users#show'
post '/users', to: 'users#create'

def helper_method(x)
  x.to_s
end
